include/mptcpd/path_manager.h

```diff
@@ -118,13 +118,15 @@ MPTCPD_API bool mptcpd_pm_ready(struct mptcpd_pm const *pm);
  *                      byte order.
  * @param[in]     id    MPTCP local address ID.
  * @param[in]     token MPTCP connection token.
+ * @param[in]     nolst Don't create listener.
  *
  * @return @c 0 if operation was successful. -1 or @c errno otherwise.
  */
 MPTCPD_API int mptcpd_pm_add_addr(struct mptcpd_pm *pm,
                                   struct sockaddr *addr,
                                   mptcpd_aid_t id,
-                                  mptcpd_token_t token);
+                                  mptcpd_token_t token,
+                                  bool nolst);
```

Comment:
This will break the API, no?

Maybe it would be safer to export a new function? `mptcpd_pm_add_addr_no_listener()`?

Internally, they will share the same code, but we don't break the API, not to force people to check the version to see what parameter to pass to a function: they can use `mptcpd_pm_add_addr_no_listener()` if it is available, and if they want to, that seems easier I think.